test/aoc.test.js

```diff
@@ -17,6 +17,9 @@ let udf_stdlib = {
   split: d => s => s.split(d),
   toNum: x => (n => Number.isNaN(n) ? undefined : n)(Number(x)),
   isLessOrEqual: (x,y) => x <= y,
+  isEqual: (x,y) => x === y,
+  exp2: x => 2 ** x,
```

Comment:
Perhaps expose this as two-argument function?

Reply:
I see. Would it be good if we do it the same way as split?
```
exp: n => x => n ** x
```